src/config/config.ts

```diff
@@ -58,6 +58,7 @@ export interface StoreLinks {
     checkoutLink: string;
     createAccountLink: string;
     forgotPasswordLink: string;
+    loginLink: string;
```

Comment:
Was this missed in Robin's PR?

Reply:
This is new. I need to know where to redirect the user when they logout.